WordPress/src/main/res/layout/fragment_me.xml

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:paddingLeft="@dimen/margin_medium"
+    android:paddingRight="@dimen/margin_medium">
+
+    <org.wordpress.android.widgets.WPNetworkImageView
+        android:id="@+id/me_avatar"
+        android:layout_width="@dimen/avatar_sz_large"
+        android:layout_height="@dimen/avatar_sz_large"
+        android:layout_alignParentTop="true"
+        android:layout_centerHorizontal="true"
+        android:layout_marginTop="@dimen/me_avatar_margin_top"
+        android:elevation="@dimen/card_elevation" />
```

Comment:
@nbradbury: Apparently you need to have a solid color background in order to have the drop shadow. I've found following thread and the bug report:

http://stackoverflow.com/questions/26572048/elevation-on-android-lollipop-not-working
https://code.google.com/p/android/issues/detail?id=78248

Since the image is rounded, if I add a background color, the background is shown as rectangle. Maybe we can add round the element and add a background color instead. I am not sure if that's the best solution though.